doc/documentation_workflow.rst

```diff
@@ -0,0 +1,152 @@
+Documentation workflow
+######################
+
+What you need to know
++++++++++++++++++++++
+
+We use **Sphinx** to generate documentation and **Read the Docs** to publish it.
+
+The documentation for NEST is contained in the ``docs`` directory within the `NEST source code repository <https://github.com/nest/nest-simulator>`_ on GitHub.
+
+Using **GitHub** allows the NEST simulator documentation to live alongside its code. Our developers benefit from smooth version control, with each release of NEST having its own documentation.
+
+.. image:: _static/img/documentation_workflow.png
+  :width: 300
+  :alt: Alternative text
+
+Changing the documentation
+++++++++++++++++++++++++++
+
+If you notice something out of place, you can submit an `issue <https://github.com/nest/nest-simulator/issues>`_ in our GitHub repository.
```

Comment:
- "out of place" strictly speaking only means docs placed incorrectly. How about "any errors or other weaknesses in the documentation"
- instead of "you can submit" better "please submit"

Reply:
Done.